tests/components/script/test_init.py

```diff
@@ -1502,3 +1502,33 @@ async def test_blueprint_script_fails_substitution(
         "{'service_to_call': 'test.automation'}: No substitution found for input blah"
         in caplog.text
     )
+
+
+async def test_responses(hass: HomeAssistant) -> None:
+    """Test we can get responses."""
+    mock_restore_cache(hass, ())
+    assert await async_setup_component(
+        hass,
+        "script",
+        {
+            "script": {
+                "test": {
+                    "sequence": {
+                        "stop": "done",
+                        "response": "5",
+                    }
+                }
+            }
+        },
+    )
+
+    assert await hass.services.async_call(
+        DOMAIN, "test", {"greeting": "world"}, blocking=True, return_response=True
+    ) == {"response": 5}
```

Comment:
This seems incorrect to me?

The response should have been the contents of the "response"? (In this case, although invalid) just `5`.

Reply:
Service calls can only return a dictionary. Since a script can return any value, we need to wrap it in a dictionary.

Reply:
Updated.